administrator/language/en-GB/com_scheduler.ini

```
@@ -102,6 +102,14 @@ COM_SCHEDULER_OPTION_ORPHANED_SHOW="Show Orphaned"
 COM_SCHEDULER_PERMISSION_TESTRUN="Test task"
```

```suggestion
COM_SCHEDULER_PERMISSION_RUN_TASK="Run task"
```

@niharikamahajan02 please check this constant and the string. You have already renamed it in the access.xml.